Libraries/LibWeb/HTML/StructuredSerialize.h

```diff
@@ -45,9 +45,9 @@ struct DeserializedRecord {
 };
 
 enum class TransferType : u8 {
-    MessagePort,
-    ArrayBuffer,
-    ResizableArrayBuffer,
+    MessagePort = 1,
```

Comment:
IMO deserves a code comment as to why we're starting at 1, to prevent this footgun in the future. (or a placeholder value for "reserved" or "unsupported").